<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rover controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonForward">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBackward">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRight">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRec">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStopRec">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSecuLow">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSecuMed">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSecuHigh">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCurrentRec">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboRec">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pressure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other sensor</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelInfos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentRec">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Currently recorded</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMov">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRec">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Security</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBck">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 cornsilk, stop:1 wheat);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listInfos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <zorder>labelBck</zorder>
   <zorder>buttonForward</zorder>
   <zorder>buttonBackward</zorder>
   <zorder>buttonLeft</zorder>
   <zorder>buttonRight</zorder>
   <zorder>buttonStop</zorder>
   <zorder>buttonRec</zorder>
   <zorder>buttonStopRec</zorder>
   <zorder>radioSecuLow</zorder>
   <zorder>radioSecuMed</zorder>
   <zorder>radioSecuHigh</zorder>
   <zorder>listCurrentRec</zorder>
   <zorder>comboRec</zorder>
   <zorder>labelInfos</zorder>
   <zorder>labelCurrentRec</zorder>
   <zorder>labelMov</zorder>
   <zorder>labelRec</zorder>
   <zorder>labelSecu</zorder>
   <zorder>listInfos</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
